src/components/App.js

```diff
@@ -59,8 +60,9 @@ const App = () => (
       <Route path="/newpassword" component={PasswordReset} />
       <Route path="/socialAuth" component={SocialAuth} />
       <Route path="/article" component={CreateArticlePage} />
-      <Route path="/articles" component={Articles} />
-      <Route path="/articles/:slug" component={SingleArticle} />
+      <Route exact path="/articles" component={Articles} />
```

Comment:
Remove exacts from the routes

Reply:
Hi @fob413. The exact was put there so that routes for viewing a specific article using its slug will be able to direct to the specific article page. If it is removed single articles would not be able to be viewed. Is there a workaround for this issue if exact is to be removed?